lib/src/notification.dart

```diff
@@ -86,13 +85,61 @@ class AndroidSettings {
     this.vibrate,
     this.vibrationPattern,
   });
+
+  AndroidSettings.fromString(Map<String, dynamic> json)
```

Comment:
Minor nitpick: Shouldn't it be called `fromJson` instead?

Reply:
How about `fromJsonString`?